tripy/docs/post0_developer_guides/how-to-add-new-ops.md

```diff
@@ -192,9 +191,9 @@ from tripy.types import ShapeLike
 # `autodoc_options` parameter.
 @export.public_api(document_under="tensor_operations")
 
-# The `convert_to_tensors` decorator automatically converts compatible
-# arguments, like `TensorLike` or `ShapeLike`s, into tensors.
-@frontend_utils.convert_to_tensors()
+# We can use the `constraints.dtypes` decorator to automatically convert
+# compatible arguments, like `TensorLike` or `ShapeLike`s, into tensors.
+@constraints.dtypes(convert_tensor_and_shape_likes=True)
```

Comment:
Can we rename/move the decorator to more accurately reflect its functionality and also to convey that we should not add more decorators if we can instead add functionality to this one? Maybe something like `@interface`?